.flake8

```
@@ -26,5 +26,4 @@ ignore=
     B008,
 
 per-file-ignores=
-    app/models/__init__.py:F401
-    app/routers/__init__.py:F401
+    **/__init__.py:F401
```

Let's add a comment for ignoring unused imports in `__init__`, especially since this is a template.
```suggestion
    # Ignore unused imports for __init__ files
    **/__init__.py:F401
```